src/lifecycle/events.js

```diff
@@ -16,7 +25,8 @@ function makeDelegateHandler (elem, handler, parsed) {
     let selector = parsed.selector;
     while (current && current !== elem.parentNode) {
       if (matches(current, selector)) {
-        e.delegateTarget = current;
+        readonly(e, 'currentTarget', current);
+        readonly(e, 'delegateTarget', elem);
```

Comment:
Yeah, it's a readonly property, but still `configurable` so in order to override the value we must re-define it.